static/js/exchange.js

```diff
@@ -13,43 +13,45 @@ const chart = Chart.render('chart', 'msgs/s')
 document.title = exchange + ' | LavinMQ'
 
 const exchangeUrl = HTTP.url`api/exchanges/${vhost}/${exchange}`
-function updateExchange () {
+function updateExchange (all) {
   HTTP.request('GET', exchangeUrl).then(item => {
     Chart.update(chart, item.message_stats)
-    let features = ''
-    features += item.durable ? ' D' : ''
-    features += item.auto_delete ? ' AD' : ''
-    features += item.internal ? ' I' : ''
-    features += item.arguments['x-delayed-exchange'] ? ' d' : ''
-    document.getElementById('e-features').textContent = features
-    document.getElementById('e-type').textContent = item.type
-    document.querySelector('#pagename-label').textContent = exchange + ' in virtual host ' + item.vhost
-    const argList = document.createElement('div')
-    Object.keys(item.arguments).forEach(key => {
-      if (key === 'x-delayed-exchange' && item.arguments[key] === false) {
-        return
-      }
-      const el = document.createElement('div')
-      el.textContent = key + ' = ' + item.arguments[key]
-      if (item.effective_arguments.includes(key)) {
-        el.classList.add('active-argument')
-        el.title = 'Active argument'
-      } else {
-        el.classList.add('inactive-argument')
-        el.title = 'Inactive argument'
+    if (all) {
+      let features = ''
+      features += item.durable ? ' D' : ''
+      features += item.auto_delete ? ' AD' : ''
+      features += item.internal ? ' I' : ''
+      features += item.arguments['x-delayed-exchange'] ? ' d' : ''
+      document.getElementById('e-features').textContent = features
+      document.getElementById('e-type').textContent = item.type
+      document.querySelector('#pagename-label').textContent = exchange + ' in virtual host ' + item.vhost
+      const argList = document.createElement('div')
```

Comment:
Instead of having an `all` argument to this function we could instead do `argList.textContent = ''` here 

Arguments shouldn't change for an exchange, but this is more that the "all" param for this functions is a little unclear

Reply:
I agree, but this is how it's done in other views 🙃 Trying to stay consistent 🤷

Reply:
Ok, makes sense .